default/generated/openjdk21/openjdk17/05-dynamic-agents/dynamic-agents.windup.yaml

```diff
@@ -0,0 +1,25 @@
+- category: potential
+  customVariables: []
+  description: |-
+    Dynamic loading of agents will be removed at a later JDK release
+    Dynamic loading of agents will be restricted at a later JDK release.. JDKs can use the `-XX:-EnableDynamicAgentLoading` command line option starting in JDK21 to ensure that agents cannot be loaded dynamically.
```

Comment:
There are multiple rule descriptions in this PR with some extra dots, for example, here:

be restricted at a later JDK release.. (two dots)

Is this something that can be manually modified? Otherwise it will be displayed in the report with these extra dots.

Thanks

Reply:
@abrugaro No, we are only converting windup xml rules to the new YAML format, we should keep them as-is without manual changes.